frontend/src/components/MapView/index.tsx

```diff
@@ -200,7 +201,7 @@ function MapView({ classes }: MapViewProps) {
   );
   const selectedLayersWithDateSupport = selectedLayers
     .filter((layer): layer is DateCompatibleLayer => {
-      if (layer.type === 'admin_level_data') {
+      if (layer.type === 'admin_level_data' || layer.type === 'static_raster') {
```

Comment:
This works, but maybe we could change the logic a bit to make it more  clear / homogeneous?

I'm thinking we could:
- add `admin_level_data` and `static_raster` to dateSupportLayerTypes
- and then return `if dateSupportLayerTypes.includes(layer.type) -> (layer.serverLayerName in serverAvailableDates || Boolean(layer.dates))

What do you think? If it's too annoying because of type conflicts don't worry about it

Reply:
@ericboucher @wadhwamatic  I tried the logic and modified a bit but did not work. Mostly related that serverLayerName is only a field for wms layer type

Reply:
the way dates are collected depend on the layer type. Most of the logic per layer is in server-utils.ts file

Reply:
ok, let's keep it like that for now then and we can revisit this later